apps/backend/migrations/20210107170456-create_groups_table.js

```diff
@@ -18,6 +18,11 @@ module.exports = {
         allowNull: false,
         defaultValue: false
       },
+      desc: {
```

Comment:
You should probably use a new migration instead, updating an old migration won't apply to existing deployments

Reply:
thanks for the review @camdenmoors